docs/reference/setup/install.asciidoc

```diff
@@ -27,6 +27,12 @@ Elasticsearch website or from our RPM repository.
 +
 <<rpm>>
 
+`docker`::
+
+An image is available for running Elasticsearch as a Docker container. It ships with https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] pre-installed and may be downloaded from the Elastic Docker Registry.
```

Comment:
In the 5.0 branch I see we have `:version:` defined as 5.0.0 and `:major_version:` as `5.x`, but no ref to the 5.0 branch. Would it be good to also define the branch as a variable so that the x-pack base URL can be composed with it?